src/controllers/commentController.js

```diff
@@ -14,12 +15,20 @@ const comment = {
     const comments = await db.Requests.findOne({
```

Comment:
We have a `RequestService` yeah? Move this database transaction to the service.

Reply:
i think this was done in refactor-develope